ui-components/src/prototypes/Card/CardFooter.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const CardFooter: React.FunctionComponent = ({ children }) => {
```

Comment:
We aren't using this typing / component style elsewhere in the app, curious about the impetus for introducing it here?

Reply:
Ah, so that's just in the example prototype which is not intended for production use…wouldn't normally write in that style.